### PR TITLE
test: enable hack/make.sh check build unit-test integration-cli-test

### PR DESCRIPTION
**Pull request by sunyuan3**
Signed-off-by: Yuan Sun <yile.sy@alibaba-inc.com>

**1.Describe what this PR did**
Create multi targets in hack/make.sh.

**2.Does this pull request fix one issue?** 
No

**3.Describe how you did it**
Add multi targets in hack/make.sh.

**4.Describe how to verify it**
hack/make.sh check
hack/make.sh build
hack/make.sh check build unit-test
hack/make.sh check build unit-test integration-cli-test

**5.Special notes for reviews**
None

### CODE REVIEW

File: hack/make.sh [hunk 1]
```diff
@@ -35,42 +26,80 @@ function install_pouch ()
...
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "make check"
+		;;
+	build)
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "make build"
+		install_pouch
+		;;
+	unit-test)
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "make unit-test"
+		;;
+	integration-cli-test)
```
Comment (Letty5411) [inline hack/make.sh]: integration-cli-test is not accurate, there are also api tests.

File: hack/make.sh [hunk 2]
```diff
@@ -35,42 +26,80 @@ function install_pouch ()
...
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "cd test && go test -c -o integration-test"
+
+		if [ $SOURCEDIR != $DIR ];then
+			[ -d $SOURCEDIR ] && rm -rf $SOURCEDIR
+			POUCHTOPDIR=`dirname $SOURCEDIR`
```
Comment (Letty5411) [inline hack/make.sh]: Please use $() instead of ``

File: hack/make.sh [hunk 3]
```diff
@@ -35,42 +26,80 @@ function install_pouch ()
...
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "make build"
+		install_pouch
+		;;
+	unit-test)
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "make unit-test"
+		;;
+	integration-cli-test)
+		docker run -ti -v `pwd`:$SOURCEDIR $IMAGE bash -c "cd test && go test -c -o integration-test"
+
+		if [ $SOURCEDIR != $DIR ];then
```
Comment (Letty5411) [inline hack/make.sh]: use [[ ]]  instead of []

File: hack/make.sh [hunk 4]
```diff
@@ -35,42 +26,80 @@ function install_pouch ()
...
+			[ ! -d $POUCHTOPDIR ] && mkdir -p $POUCHTOPDIR
+			ln -sf $DIR/ $SOURCEDIR
 		fi
-	done
 
-	echo "verify pouch version"
-	pouch version
+		#start pouch daemon
+		echo "start pouch daemon"
+		pouchd > $TMP/log 2>&1 &
```
Comment (Letty5411) [inline hack/make.sh]: do we need to use "pouchd -D"?

**Comment by Letty5411** [comment]
others LGTM

**Comment by allencloud** [comment]
LGTM

**Pull request merged**